Fatal Python error: init_import_site: Failed to import the site module
Python runtime state: initialized
Traceback (most recent call last):
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1176, in exec_module
  File "<frozen site>", line 635, in <module>
  File "<frozen site>", line 621, in main
  File "<frozen site>", line 361, in addusersitepackages
  File "<frozen site>", line 342, in getusersitepackages
  File "<frozen site>", line 331, in getuserbase
  File "<frozen site>", line 305, in _getuserbase
  File "<frozen site>", line 295, in joinuser
  File "<frozen posixpath>", line 71, in join
KeyboardInterrupt
